### ISSUE TITLE
How to prevent fuel cans from being destroyed when empty?

### ISSUE THREAD

**Issue by ubergeek77** [FlansMods/FlansMod#843]
I suppose this is a feature/enhancement request of sorts.

I currently have the base Flan's Mod + Simple Parts Pack + Ye Olde Pack. Right now, when a fuel can is used to fill up the tank of a Biplane/Two-Seat Biplane, the fuel can will be destroyed when (happens when damage is at 1000). I would like to find a way to prevent the fuel can from being destroyed at all; when the fuel is depleted, I'd like for the fuel can to stay as-is at damage:1000. The reason for this because I am working on a personal mod pack, where I have used MineTweaker to change the recipe of the fuel can to better work with other FTB mods (such as the ability to craft an empty/damage = 1000 fuel can using Plastic from MineFactory Reloaded, and filling the empty fuel can with Fuel from BuildCraft). I have it integrated well for my personal set of mods, and the last thing to do is make sure that Flan's Mod doesn't destroy the empty fuel can item.

If there is a simple config/txt file setting for this, I haven't seen it. More likely, this will require a modification to source. I've noticed that I may be able to change `FlansMod/src/main/java/com/flansmod/common/tools/ToolType.java:33` to "false," but I am unsure if this would even change the behavior I'm looking for (since the fuel can is a "Part" and not a "Tool"), and on the flipside, I'd like to keep the behavior of all the other tools the same.

Even if such a change is against the wishes of the Flan's Mod maintainers, it would be awesome if someone could point me in the right direction in the source so I can compile such a change myself.

TL;DR How can I stop fuel cans from being destroyed when damage = 1000?

Thanks for the help!
